openmc/deplete/results_list.py

```diff
@@ -192,3 +196,100 @@ def get_depletion_time(self):
         for ix, res in enumerate(items):
             times[ix] = res.proc_time
         return times
+
+    def get_times(self, time_units="d") -> np.ndarray:
+        """Return the points in time that define the depletion schedule
+
+        Parameters
+        ----------
+        time_units : {"s", "d", "h", "min"}, optional
+            Return the vector in these units. Default is to
+            convert to days
+
+        Returns
+        -------
+        numpy.ndarray
+            1-D vector of time points
+
+        """
+        check_type("time_units", time_units, str)
+
+        times = np.fromiter(
+            (r.time[0] for r in self),
+            dtype=self[0].time.dtype,
+            count=len(self),
+        )
+
+        if time_units == "d":
+            times /= (60 * 60 * 24)
+        elif time_units == "h":
+            times /= (60 * 60)
+        elif time_units == "min":
+            times /= 60
+        elif time_units != "s":
+            raise ValueError(
+                f'Unable to set "time_units" to {time_units} since it is not '
```

Comment:
Right now, our stated support is [Python 3.5+](https://docs.openmc.org/en/latest/usersguide/install.html#usersguide-python-prereqs), although I'm in favor of moving to Python 3.6+ given that 3.5 is already past EOL and it's not even in our test matrix anymore (and man is it nice to have f-strings!). That being said I wanted to give others a chance to chime in here, especially @openmc-dev/technical-committee.

Reply:
Oh yeah that totally slipped my mind. f-strings are just so nice! But not really worth dropping py 3.5 *just* for f-strings. I'm fine to change this back to fit with 3.5, but I do think a discussion with the technical committee would be fruitful.

Maybe for 0.13.0 release? I see 0.12.1 is listed as a milestone

Reply:
f-strings have been removed